contributors/guide/pull-requests.md

```diff
@@ -505,38 +476,37 @@ further help explain the _"why"_ the commit was made.
 - [A Note About Git Commit Messages - Tim Pope](https://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html)
 
 
-## 8. KISS, YAGNI, MVP, etc.
+## KISS, YAGNI, MVP, etc.
 
 Sometimes we need to remind each other of core tenets of software design - Keep
-It Simple, You Aren't Gonna Need It, Minimum Viable Product, and so on. Adding a 
-feature "because we might need it later" is antithetical to software that ships. 
+It Simple, You Aren't Gonna Need It, Minimum Viable Product, and so on.
+Adding a feature "because we might need it later" is antithetical to software that ships. 
 Add the things you need NOW and (ideally) leave room for things you might need 
 later - but don't implement them now.
```

Comment:
Hm, what do you think of putting this and the items below above the commit msg guidelines?  I'm sort of leaning towards before it, but could go the other way too if you think thats better formatted.